### PR TITLE
Understanding DHCP Server  Router Configuration  

### LINKED ISSUE #2689: Understanding DHCP Server Router Configuration
### Proposed title of article
Understanding how to configure DHCP server on routers.

### Introduction paragraph (2-3 paragraphs):
A dedicated DHCP server is scalable and easy to manage but it can be costly to have one at every location in a network, more so when the network is large. In such situations, network routers can be configured to provide the DHCP services without the need for a dedicated server. This will enable the network administrator to automate the process of configuring devices on the IP networks by use of DHCP protocol and save the extra cost of buying  DHCP servers.

In this article, we will show the reader how one can utilize a router to provide  DHCP services and be able to configure it on the network. This will reduce the workload of assigning IP addresses manually on the network devices.

### Key takeaways: 
-Understand how the DHCP server works.
-Understand how to configure a router to provide DHCP services.
-How to use DHCP protocols in assigning IP addresses dynamically using DHCP server

### References: 
N/A

### Templates to use as guides
N/A

### PR DESCRIPTION
**Pull request by samotieno**
### Article checklist

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #2689 "](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**
   This fixes #2689   
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(2689 #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**

fixes #2926

### CODE REVIEW

**Comment by hectorkambow** [comment]
@samotieno 
Good afternoon and thank you for submitting your article for publishing.
Your article has been entered into our queue and should be reviewed (and published pending any changes) as soon as a peer reviewer is finished with the ones in the queue before it. 🚀

Thank you.

**Comment by samotieno** [comment]
cc @louisefindlay23

**Comment by louisefindlay23** [comment]
Thanks for being so patient, @samotieno. I'm not an expert in networking so have forwarded this to one of our peer reviewers who is so they can give both a preliminary and technical review.

Hi, @collins-a. Are you free to review this article?

**Comment by collins-a** [comment]
> Thanks for being so patient, @samotieno. I'm not an expert in networking so have forwarded this to one of our peer reviewers who is so they can give both a preliminary and technical review.
> 
> Hi, @collins-a. Are you free to review this article?

Hey @louisefindlay23, no problem taking this on. I'll carry out all the reviews on articles assigned to me over the weekend.

**Comment by samotieno** [comment]
> > Thanks for being so patient, @samotieno. I'm not an expert in networking so have forwarded this to one of our peer reviewers who is so they can give both a preliminary and technical review.
> > Hi, @collins-a. Are you free to review this article?
> 
> Hey @louisefindlay23, no problem taking this on. I'll carry out all the reviews on articles assigned to me over the weekend.

ookk @collins-a

File: content/articles/dhcp-server-router-configuration/index.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+As we have seen, network routers can save the organization the cost of buying dedicated servers by providing the DHCP server functionality when configured properly. This information is crucial and will equip the learner with the knowledge of how to configure the routers to eliminate the burden of statically assigning IP addresses.
+
+
+To summarize:
+- We learned what a DHCP server is.
+- We explored how to use a router to act as a DHCP server.
+- We configured a network, verified and configured the routers to perform the DHCP server functions.
+- We learned how to enable and disable the DHCP server on the network.
+
+One can find more information about  DHCP-router configuration [here](https://contenthub.netacad.com/srwe/7.2.1).
```
Comment (collins-a) [inline content/articles/dhcp-server-router-configuration/index.md:201]: At the moment, this link is not working. Confirm if it works on your end or give an alternate link

File: content/articles/dhcp-server-router-configuration/index.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+Happy coding.
```
Comment (collins-a) [inline content/articles/dhcp-server-router-configuration/index.md:203]: Consider adding a couple of references or additional material for the reader to get more value even after reading your tutorial.

**Review by collins-a — APPROVED** [review]
@samotieno good tutorial on configuring routers to handle DHCP. The processes and images are well done. Look at the handful of comments I've made and we'll be good to go.

File: content/articles/dhcp-server-router-configuration/index.md (hunk 1 above)
Comment (samotieno) [reply content/articles/dhcp-server-router-configuration/index.md]: @collins-a done with changes  checkout

**Comment by hectorkambow** [comment]
plagiarism check done

**Review by louisefindlay23 — APPROVED** [review]
Great work, @samotieno. Final review complete. Be sure to look over the minor edits I made to inform future articles.

Thanks, @collins-a for peer reviewing.

**Pull request merged**